docs/metrics.md

```diff
@@ -0,0 +1,14 @@
+# Build Controller Metrics
+
+The Build component exposes several metrics to help you monitor the health and behavior of your build resources.
+
+Following build metrics are exposed at service `build-operator-metrics` on port `8383`.
+
+| Name | Type | Description | Status |
+| ---- | ---- | ----------- | ------ |
+| `up_status` | Gauge | The up status of the build-controller. | experimental |
```

Comment:
Prometheus has an automatic `up` metric that indicates whether a target can be scraped. This is typically used for "up" monitoring.

Reply:
I didn't find this up metric, is it also related to our build controller `up` status? Or a general up status?

Reply:
I don't know how you set this metric, but generally speaking you want to avoid things like this. You want to instead expose raw data and make the decision of "up" or "down" based on the raw data and not create what is essentially a bespoke monitoring system in your component.

Reply:
Hi @adambkaplan ,
How can we get the `version info` in the metric or the code logic?

Reply:
I compared other community metric designs, there is no up_status.

So let me remove it in this initial PR first.

I removed in the latest commit: https://github.com/redhat-developer/build/pull/235/commits/248b3db1c98c9c36a87cab7f6f75f2095cb8400a